Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	// TODO hartez 2018/06/23 13:42:22 Trying this out for a nicer read in OnElementPropertyChanged, not sure if I like it yet	
+	public static class PropertyChangedEventArgsExtensions
```

Comment:
change this to internal while we are unsure

Reply:
I can't, the renderers need visibility to it.

Reply:
If this is a helper specific to renderers but that we don't want to expose publicly, we can copy the helper class in every platform project.

Reply:
Okay, moved to internal helper classes on each platform.